hyperspy/io_plugins/digital_micrograph.py

```diff
@@ -819,68 +819,74 @@ def _parse_string(self, tag):
             return tag
 
     def get_mapping(self):
-        is_scanning = "DigiScan" in self.imdict.ImageTags.keys()
+        if 'source' in self.imdict.ImageTags.keys():
+            # For stack created with the stack builder plugin
+            tags_path = 'ImageList.TagGroup0.ImageTags.source.Tags at creation'
+            ImageTags_dict = self.imdict.ImageTags.source['Tags at creation']
```

Comment:
I would avoid using camelcase for variables as it makes them look like classes, making the code harder to read.